pkg/services/ngalert/notifier/channels/line.go

```diff
@@ -21,6 +21,10 @@ var (
 
 // NewLineNotifier is the constructor for the LINE notifier
 func NewLineNotifier(model *NotificationChannelConfig, t *template.Template) (*LineNotifier, error) {
+	if model.Settings == nil {
```

Comment:
Wherever we are using `model.DecryptedValue`, you also want to check `.SecureSettings` for a `nil` :)